src/5e-SRD-Monsters.json

```diff
@@ -29589,7 +29589,7 @@
             {
               "name": "Suggestion",
               "level": 2,
-              "url": "/api/spells/mass-suggestion",
+              "url": "/api/spells/suggestion",
```

Comment:
It's actually neither of these and is `shillelagh`

Reply:
Just checked the SRD 5.1 for Rakshasa, it lists the following:

> * At will: detect thoughts, disguise self, mage hand, minor illusion
> * 3/day each: charm person, detect magic, invisibility, major image, **suggestion**
> * 1/day each: dominate person, fly, plane shift, true seeing

The rest of the spells look correct.

Reply:
For some reason I thought it was Dryad. My bad.